### ISSUE TITLE
Can't generate a working form or crud for an entity named the same as a built-in type

### ISSUE THREAD

**Issue by electrolinux** [sensiolabs/SensioGeneratorBundle#500]
### Use case
I just need an entity named Url.

### How to reproduce:
````bash
symfony new foo 2.8
cd foo
# using pdo_sqlite
sed -i 's!    # database_path: "%kernel.root_dir%/data.db3"!    database_path: "%kernel.root_dir%/data.db3"!' app/config/parameters.yml.dist
echo "    database_path: '%kernel.root_dir%/data.db3'" >> app/config/parameters.yml
sed -i 's/pdo_mysql/pdo_sqlite/' app/config/config.yml
sed -i 's/        #     path:     "%database_path%"/        path:     "%database_path%"/' app/config/config.yml
app/console doctrine:generate:entity --entity=AppBundle:Url \
    --format=annotation --fields="href:string(255)"  \
    --no-interaction
app/console doctrine:generate:entities AppBundle:Url
app/console doctrine:generate:crud --entity=AppBundle:Url \
    --with-write --route-prefix=url --no-interaction
app/console server:run
````

So far so good, but going to /url/new throw an Exception, because the generated AppBundle/Form/UrlType.php is missing a getBlockPrefix() method.

### Solution (for 2.8+):
generate a getBlockPrefix method when the Entity name is also one of the built-in types.

Not tested with 2.6, but I suspect that a getName() method would also be required.

I'll be happy to provide a pull request, but it could be easier (and quicker) for anybody else well knowing the symfony code to do so :)

**Comment by javiereguiluz** [comment]
@electrolinux I tried to run your commands to reproduce the error, but I got some errors:

```
$ app/console doctrine:generate:entities AppBundle:Url

[Doctrine\ORM\ORMException]
  Unknown Entity namespace alias 'AppBundle'.


...



$ app/console doctrine:generate:crud --entity=AppBundle:Url \
     --with-write --route-prefix=url --no-interaction

 [RuntimeException]
  Entity "Url" does not exist in the "AppBundle" bundle.
Create it with the "doctrine:generate:entity" command and
then execute this command again.
```

**Comment by electrolinux** [comment]
@javiereguiluz Yes, I've seen such problem when running all commands in a bash script. In such a case, a little pause seems necessary between each app/console invocation. 
I've created a gist with a bash script generating the demo:
https://gist.github.com/electrolinux/45dbf07a214901bbbdfa15357d817e5b
Just put a copy (or symlink) of a composer.phar beside the script and run the script.

I've just tested now with the latest 2.8.8
